### PR TITLE
Fix clippy warning which got accidentally pushed to main

### PR DESCRIPTION
**Pull request by Jauler**
Reverts NordSecurity/libtelio#566 As it broke some clippy checks

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by mathiaspeters — DISMISSED** [review]

**Review by LukasPukenis — APPROVED** [review]

File: crates/telio-utils/src/repeated_actions.rs [hunk 1]
```diff
@@ -114,6 +114,7 @@ where
             .map(|(key, interval, action)| {
                 interval
                     .map(move |instant| {
+                        dbg!(instant);
                         (key, action)
                     })
```
Comment (tomaszklak) [inline crates/telio-utils/src/repeated_actions.rs:119]: I think it's enough to change line `move |instant| {` to `move |_| {`.

Comment (matszczygiel) [reply crates/telio-utils/src/repeated_actions.rs:119]: Yeah, that should do the trick

Comment (Jauler) [reply crates/telio-utils/src/repeated_actions.rs:119]: You are right, lets do that :D

**Review by tomaszklak — APPROVED** [review]
+2 whichever way you choose :)

File: crates/telio-utils/src/repeated_actions.rs (hunk 1 above)
Comment (Jauler) [reply crates/telio-utils/src/repeated_actions.rs]: done

**Review by tomaszklak — APPROVED** [review]

**Review by tomaszklak — APPROVED** [review]

**Pull request merged**